units/MODULE.md

```diff
@@ -1,3 +1,3 @@
-# Module Units
+# Module units
```

Comment:
drive-by fix: name here needs to match the module name exactly for the contents to be used by dokka